my-menu-tsps-web/pom.properties

```diff
@@ -0,0 +1,4 @@
+#Generated by Apache Maven ${maven.version}
+version=${project.version}

```

Comment:
Weird: this has been committed with Windows-style CR-LR end-of-lines (it's marked in red by GitHub `^M`). You should probably fix this.